src/components/blocks/image.ts

```diff
@@ -16,20 +20,38 @@ interface Image {
 
 // ----- Functions ----- //
 
+const imageRatioStyles = (sizes = "100vw", width: number, height: number): SerializedStyles => css`
+        --size: ${sizes};
+        height: calc(var(--size) * ${height / width});
+        background: ${palette.neutral[97]};
+
+        ${from.wide} {
+            height: calc(620px * ${height / width});
+        }
+    `
+
 const isImage = (elem: BlockElement): boolean =>
   elem.type === 'image';
 
 const element = (sizes: string) => 
-    (alt: string, assets: AssetUtils.Asset[], salt: string): ReactNode =>
-        h('img', {
-            sizes,
-            srcSet: AssetUtils.toSrcset(salt, assets).withDefault(''),
-            alt,
-            src: AssetUtils.toUrl(salt, assets[0]),
-        });
+    (alt: string, assets: AssetUtils.Asset[], salt: string): ReactNode => {
+        const masterAsset = assets.find(asset => asset.typeData.isMaster);
+        const aspectRatio: number[] = fromNullable(masterAsset)
+            .map(asset => [asset.typeData.height, asset.typeData.width])
+            .withDefault([3, 5])
+        const [height, width] = aspectRatio;
+
+            return styledH('img', {
+                css: imageRatioStyles(sizes, width, height),
+                sizes,
+                srcSet: AssetUtils.toSrcset(salt, assets).withDefault(''),
+                alt,
+                src: AssetUtils.toUrl(salt, assets[0]),
+            });
+    }
 
 const immersiveImageElement = element('calc(80vh * 5/3)');
-const imageElement = element('100%');
+const imageElement = element('100vw');
```

Comment:
Does `100%` no longer work here?

Reply:
It would collapse the height of the placeholder to 0